js/modules/loki_app_dot_net_api.js

```diff
@@ -109,17 +109,27 @@ class LokiAppDotNetServerAPI {
       JSON.stringify(objToSign)
     );
     */
+    let pName = profileName;
```

Comment:
```suggestion
    let pName = profileName || '';
```
This would also work